airbyte-cdk/python/airbyte_cdk/sources/connector_state_manager.py

```diff
@@ -139,7 +141,7 @@ def _extract_from_state_message(
                     name=per_stream_state.stream.stream_descriptor.name, namespace=per_stream_state.stream.stream_descriptor.namespace
                 ): per_stream_state.stream.stream_state
                 for per_stream_state in state
-                if per_stream_state.type == AirbyteStateType.STREAM and hasattr(per_stream_state, "stream")
+                if per_stream_state.type == AirbyteStateType.STREAM and hasattr(per_stream_state, "stream")  # type: ignore
```

Comment:
What are we ignoring here? Is it because if we assume `per_stream_state` as the `stream` attribute, it will have the `type` as well?